src/prompt-input/styles.scss

```diff
@@ -0,0 +1,98 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+/* stylelint-disable */
+
+@use '../internal/styles' as styles;
+@use '../internal/styles/tokens' as awsui;
+
+$send-icon-right-spacing: awsui.$space-static-xxs;
+
+.root {
+  position: relative;
+}
+
+.textarea {
+  @include styles.styles-reset;
+  // Restore browsers' default resize values
+  resize: none;
+  // Restore default text cursor
+  cursor: text;
+  // Allow multi-line placeholders
+  white-space: pre-wrap;
+
+  padding-block: styles.$control-padding-vertical;
+  padding-inline: styles.$control-padding-horizontal;
+
+  color: awsui.$color-text-body-default;
+  max-inline-size: 100%;
+  inline-size: 100%;
+  display: block;
+  box-sizing: border-box;
+  background-color: awsui.$color-background-input-default;
+  border-start-start-radius: styles.$control-border-radius;
+  border-start-end-radius: styles.$control-border-radius;
+  border-end-start-radius: styles.$control-border-radius;
+  border-end-end-radius: styles.$control-border-radius;
+  border-block: styles.$control-border-width solid awsui.$color-border-input-default;
+  border-inline: styles.$control-border-width solid awsui.$color-border-input-default;
+  @include styles.font-body-m;
+
+  &.textarea-readonly {
+    @include styles.form-readonly-element;
+  }
+
+  &::placeholder {
+    @include styles.form-placeholder;
+    opacity: 1;
+  }
+
+  &:-ms-input-placeholder {
+    @include styles.form-placeholder;
+  }
+
+  &:focus {
+    @include styles.form-focus-element;
+  }
+
+  &:invalid {
+    // discard built-in invalid styles, customers should use `invalid` property only (AWSUI-3947)
+    box-shadow: none;
+  }
+
+  &:disabled {
+    @include styles.form-disabled-element;
+    cursor: default;
+
+    &::placeholder {
+      @include styles.form-placeholder-disabled;
+      opacity: 1;
+    }
+
+    &:-ms-input-placeholder {
+      @include styles.form-placeholder-disabled;
+    }
+  }
+
+  &.textarea-invalid {
+    @include styles.form-invalid-control();
+  }
+
+  &.textarea-warning {
+    @include styles.form-warning-control();
+  }
+  &.textarea-with-button {
+    padding-inline-end: calc(styles.$control-padding-horizontal + $send-icon-right-spacing + awsui.$size-icon-normal);
+  }
+}
+
+.button {
+  position: absolute;
+  inset-inline-end: $send-icon-right-spacing;
+  inset-block-end: 0;
+}
+
+.action-button {
+  /* used in test-utils */
```

Comment:
general thought: we were discussing separation of classes for tests from styling-related to improve stability of selectors. That was done for [copy-to-clipboard](https://github.com/cloudscape-design/components/blob/fcd9ab11a71083f7476ba2c5f0a494ef7c047398/src/copy-to-clipboard/test-classes/styles.scss#L4) and [anchor navigation](https://github.com/cloudscape-design/components/blob/fcd9ab11a71083f7476ba2c5f0a494ef7c047398/src/anchor-navigation/test-classes/styles.scss#L4). I'd suggest we continue doing so for new components